master.css

```diff
@@ -13,6 +13,18 @@ body {
 }
 table {
     margin-bottom: 2em; width: 100%;
+    border-spacing: 0;
+}
+thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
```

Comment:
For the record — https://bugzilla.mozilla.org/show_bug.cgi?id=975644 Looks like that ticket is _just about_ to be fixed. But you never know.